src/uu/factor/src/cli.rs

```diff
@@ -34,14 +33,21 @@ fn print_factors_str(num_str: &str, w: &mut impl io::Write) -> Result<(), Box<dy
 }
 
 pub fn uumain(args: impl uucore::Args) -> i32 {
-    let matches = app!(SYNTAX, SUMMARY, LONG_HELP).parse(
-        args.collect_str(InvalidEncodingHandling::Ignore)
-            .accept_any(),
-    );
+    let matches = App::new(executable!())
+        .version(VERSION)
+        .about(SUMMARY)
+        .arg(Arg::with_name("NUMBER").multiple(true))
+        .get_matches_from(args);
     let stdout = stdout();
     let mut w = io::BufWriter::new(stdout.lock());
 
-    if matches.free.is_empty() {
+    if let Some(values) = matches.values_of("INPUT") {
```

Comment:
Seems like I did not submit my comment in the code :). I think the problem is that it should be `"NUMBER"` instead of `"INPUT"`

Reply:
Yeah, last minute refactoring bit me :x
Fixed it now, thanks :)